### DISCUSSION TITLE
Seeking Information on Enterprise Adoption of Pekko

### DISCUSSION THREAD

**Discussion by sathees17** [apache/pekko#1482] [General]
Hi Pekko Community,

We are planning to use Pekko for an upcoming enterprise project and would like to know if there are any other enterprise-level projects that have already adopted Pekko.

Additionally, we would appreciate any advice or insights you can provide on using Pekko for enterprise projects, especially in terms of stability, performance, and long-term support.

Thank you in advance for your guidance!

**Comment by pjfanning** [comment]
I'll leave the discussion open for users to comment on their experience of using Apache Pekko.
Many of our jars get 6 figure downloads per month - the totals across all jars are well into the millions. Of course, that is no indication of how many users use Pekko in production - and many users are using Pekko indirectly as a transitive dependency of applications like Play, Apache Flink, Tapir, etc.

In terms of Long Term Support, the community health of Apache Pekko is good. The community is small but active. The code base is quite old and battle tested if you think of Apache Pekko as being based on stable releases of Akka from before their license changes (about 2 years ago).

Still, Apache Pekko is not backed by a corporation offering support contracts. We might see Pekko developers offering some sort of paid support in the future - not as part of Apache Pekko directly but as independent developers.

**Comment by He-Pin** [comment]
We at alibaba/taobao are using Pekko now, where is was Akka-stream.
You can take a look at Taobao English, search it on Twitter/X where every page go through a Pekko stream.
